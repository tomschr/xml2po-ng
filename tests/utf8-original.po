msgid ""
msgstr ""
"Project-Id-Version: something\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2005-04-02 11:37+0200\n"
"PO-Revision-Date: 2005-04-01 22:51+0300\n"
"Last-Translator: Pauli Virtanen <pauli.virtanen@hut.fi>\n"
"Language-Team: Finnish <translation-team-fi@lists.sourceforge.net>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: utf8-original.xml:3 (p) utf8-original.xml:5
msgid "This is an untranslated message, with UTF-8 characters: åäö ÅÄÖ."
msgstr "A translated message (for a msgid with utf-8)."

#: utf8-original.xml:4 (p) utf8-original.xml:6
msgid "This is an untranslated message, with no UTF-8 characters."
msgstr "A translated message (for a msgid with NO utf-8)."

#. Put one translator per line, in the form of NAME <EMAIL>, YEAR1, YEAR2
#: utf8-original.xml:0 (None)
msgid "translator-credits"
msgstr "Yeah."
