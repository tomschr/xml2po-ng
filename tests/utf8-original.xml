<?xml version="1.0" encoding="utf-8"?>
<test>
<p>This is an untranslated message, with UTF-8 characters: åäö ÅÄÖ.</p>
<p>This is an untranslated message, with no UTF-8 characters.</p>
<p>This is an untranslated message, with UTF-8 characters: åäö ÅÄÖ.</p>
<p>This is an untranslated message, with no UTF-8 characters.</p>
<</test>
