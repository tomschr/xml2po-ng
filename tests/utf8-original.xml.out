<?xml version="1.0" encoding="utf-8"?>
<test>
<p>A translated message (for a msgid with utf-8).</p>
<p>A translated message (for a msgid with NO utf-8).</p>
<p>A translated message (for a msgid with utf-8).</p>
<p>A translated message (for a msgid with NO utf-8).</p>
</test>
